### ISSUE TITLE
Automatically update views when a stream is created 

### ISSUE THREAD

**Issue by msach22** [opentok/cordova-plugin-opentok#136]
# Feature Request

**Description**
> (Include problem, use cases, benefits, and/or goals)
Currently, Ionic applications have to manually call `OT.updateViews` when they subscribe to a stream.
**Proposal**
We can detect if it's an Ionic application and call `OT.updateViews` automatically so the application layer does not have to do it.

**Links / references**

**Comment by mark-veenstra** [comment]
The PR https://github.com/opentok/cordova-plugin-opentok/pull/59 is already implementing such a feature

**Comment by msach22** [comment]
Yes, but we can't merge #59 in because it drops support for some Android platforms that OpenTok still supports.

**Comment by Shtibel** [comment]
when will this be updated?

**Comment by Shtibel** [comment]
Hi @msach22 ,
So now we don't need to do OT.updateViews anymore?

**Comment by msach22** [comment]
@Shtibel `OT.updateViews` is only called by the plugin when a stream is created, not when there are changes in the DOM.

Currently, we are unable to add it when there are all DOM changes because we still plan on supporting Android 16.

**Comment by Shtibel** [comment]
Ok thank you @msach22.

**Comment by msach22** [comment]
I'l update you if we can do things without dropping support!
